lib/File/Find/Rule.pm

```diff
@@ -563,11 +576,11 @@ sub in {
         my $discarded;
         return unless ' . $fragment . ';
         return if $discarded;
-        if ($relative) {
-            push @found, $relpath if $relpath ne "";
+        if ($relative and ($relpath ne "") ) {
```

Comment:
This has changed the structure and behaviour of this code.

Written out longhand the behaviour was:

```
if ($relative) {
  if ($relpath ne "") {
    push @found, $relpath;
  } else {
    # no pushing happens
  }
} else {
  push @found, $path;
}
```

Whereas with your change it's going to ignore $relative.